# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/PBrowser.ipdl
# Commit: ea966e78b296
# Full Hash: ea966e78b296be37881d4fa36f7cc805e7bbac82
# Author: Nika Layzell <nika@thelayzells.com>
# Date: 2020-04-07 03:53:09
# Regressor Bug: 1616353
# File Overlap Count: 1
# Description:
#   Bug 1616353 - Part 6.3: Replace many nsILoadContext mutators with assertions, r=smaug
#   
#   Many of these setters are redundant, and duplicate work done elsewhere. As it is
#   no longer correct to set nsILoadContext attributes at these times, they're being
#   replaced with assertions.
# ==============================================================================

diff -r cb88e0bbb3b9 -r ea966e78b296 dom/ipc/PBrowser.ipdl
--- a/dom/ipc/PBrowser.ipdl	Mon Apr 06 14:29:50 2020 +0000
+++ b/dom/ipc/PBrowser.ipdl	Mon Apr 06 14:29:52 2020 +0000
@@ -1014,12 +1014,6 @@
     async AllowScriptsToClose();
 
     /**
-     * Tell the BrowserChild what OriginAttributes it should inherit from. This must
-     * be called before the first non-blank document is loaded in the BrowserChild.
-     */
-    async SetOriginAttributes(OriginAttributes aOriginAttributes);
-
-    /**
      * Pass the current handle for the current native widget to the content
      * process, so it can be used by PuppetWidget.
      */